providers/apis/defi/raydium/types.go

```diff
@@ -0,0 +1,74 @@
+package raydium
```

Comment:
can you add a default api and market config for the provider?

Reply:
I'll make this in a separate PR?